service-tests/waffle.js

```diff
@@ -0,0 +1,58 @@
+const Joi = require('joi');
+const ServiceTester = require('./runner/service-tester');
+
+const t = new ServiceTester({ id: 'waffle', title: 'Waffle.io' });
+module.exports = t;
+
+
+/*
+ALL test against fake data (`fakeData`).
```

Comment:
How come this approach was taken? The most helpful comments answer that question. 😀 

Is there any reason not to include a live test as well, so if the API changes we find out about the problems first?

Reply:
Actually, there  have some simple counting ( using 2 `for`  loop) logic, so I though, It would be nice if I write test (1st one) for that logic. 

Okay, I am changing the 2nd test against live test. :)